source/production/F0.Minesweeper.Server/Pages/Index.razor.cs

```diff
@@ -1,9 +1,23 @@
+using F0.Minesweeper.Components;
 using F0.Minesweeper.Components.Abstractions;
+using F0.Minesweeper.Components.Abstractions.Enums;
 
 namespace F0.Minesweeper.Server.Pages
 {
 	public partial class Index
 	{
 		public MinefieldSize MinefieldSize = new MinefieldSize(10, 10);
+
+		public Minefield? Minefield1 { get; set; }
```

Comment:
I think it's unnecessary to make this `public` for now since it's only used here. So we could actually make it `private`

Reply:
Sure. I think we will move this control from here. But, I will keep it in mind for the next time!